app/contributing/markdown-rules.md

```diff
@@ -444,18 +375,22 @@ you can use the indent filter on a capture of your tabs.
 {% capture the_code %}
 {% navtabs codeblock %}
 {% navtab cURL %}
-<div class="copy-code-snippet"><pre><code>curl -i -X POST http://<div contenteditable="true">{HOSTNAME}</div>:8001/event-hooks \
+```sh
```

Comment:
something is off with these navtabs, they are not being rendered right

Reply:
I found out what the problem is
now that each tab has a code block in it, the parser doesn't know where they start/end.
we need to add another ` to the `{% raw %}` block, [here](https://github.com/Kong/docs.konghq.com/pull/5956/files#diff-531b7ce7eac5adee79702a5d9130a418ef63b625d41e3134e4e83837b8669648L443) and [here](https://github.com/Kong/docs.konghq.com/pull/5956/files#diff-531b7ce7eac5adee79702a5d9130a418ef63b625d41e3134e4e83837b8669648L463) so that the parser knows that it needs to treat everything inside it as different code blocks

Reply:
Fixed!